### PR TITLE
ES requirements check on Pantheon

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

File: web/modules/custom/server_search/server_search.module [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  // Define the path to the JSON file.
+  $json_file_path = $base_private_dir . '/' . Settings::get('site_id') . '.es.secrets.json';
+
+  // Check if JSON file exists in the "config/elasticsearch" directory.
+  if (!file_exists($json_file_path)) {
+    // If JSON file does not exist, set requirement error.
+    $requirements['server_search_json'] = [
+      'title' => t('Server Search JSON'),
+      'value' => t('Not found'),
+      'description' => t('The required JSON file does not exist in the "config/elasticsearch" directory.'),
```
Comment (amitaibu) [inline web/modules/custom/server_search/server_search.module:36]: Maybe print the name of the expected file

**Comment by AronNovak** [comment]
image

**Comment by AronNovak** [comment]
image

**Comment by AronNovak** [comment]
image

**Comment by AronNovak** [comment]
image

File: web/modules/custom/server_search/server_search.install [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    return $requirements;
+  }
+
+  // If JSON file exists, check if it has the required keys.
+  $json_data = json_decode(file_get_contents($json_file_path), TRUE);
+
+  $required_keys = array_unique([
+    'qa',
+    'live',
```
Comment (amitaibu) [inline web/modules/custom/server_search/server_search.install:57]: let's move`live` after `test`

**Review by amitaibu — APPROVED** [review]
Thanks

File: web/modules/custom/server_search/server_search.install (hunk 2 above)
Comment (AronNovak) [reply web/modules/custom/server_search/server_search.install]: done

**Pull request merged**